src/frontends/pytorch/src/transforms/prim_list_unpack_replacer.cpp

```diff
@@ -173,6 +173,32 @@ PrimListUnpackReplacer::PrimListUnpackReplacer() {
             }
         }
 
+        if (auto broadcast_tensors = cast_fw_node(input_node, "aten::broadcast_tensors")) {
+            auto tensors = cast_fw_node(broadcast_tensors->input_value(0).get_node_shared_ptr(), "prim::ListConstruct");
+            if (!tensors) {
+                add_exception_to_fw_node(input_node,
+                                         "aten::broadcast_tensors: only prim::ListConstruct supported as input.");
+                return false;
+            }
+            auto zero = opset10::Constant::create(element::i32, Shape{}, {0});
+            Output<Node> final_shape_t = zero;
+            for (auto input : tensors->inputs()) {
+                auto tensor_shape = std::make_shared<opset10::ShapeOf>(input.get_source_output());
+                auto zero_broadcasted =
+                    std::make_shared<opset10::Broadcast>(zero, tensor_shape, ov::op::BroadcastType::BIDIRECTIONAL);
+                final_shape_t = std::make_shared<opset10::Add>(final_shape_t, zero_broadcasted);
```

Comment:
As far as I see you use `Add` here with zero to broadcast many shapes to broadcasted zero, meaning you use 2 broadcast ops, can it be 1 broadcast op?

Reply:
I think it is not possible as you do not know maximal shape before starting broadcasting, but I replaced first broadcast with just add